ILSpy/LoadedAssembly.cs

```diff
@@ -55,6 +55,8 @@ public LoadedAssembly(AssemblyList assemblyList, string fileName, Stream stream
 
 			this.assemblyTask = Task.Factory.StartNew(LoadAssembly, stream); // requires that this.fileName is set
 			this.shortName = Path.GetFileNameWithoutExtension(fileName);
+			this.resolver = new MyAssemblyResolver(this);
+			this.universalResolver = new MyUniversalResolver(this);
```

Comment:
This seems to cause ILSpy to crash on startup if `ILSpy.xml` contains a PE file that is not a managed assembly.